lib/elements/dom-repeat.html

```diff
@@ -456,13 +456,19 @@
     }
 
     __handleObservedPaths(path) {
-      if (this.__observePaths) {
```

Comment:
Previously the condition was only on `this.__observePaths`. Is there a case where `this.__sortFn` and `this.__filterFn` are both falsy, but `this.__observePaths` is truthy? In that case, this would regress, as the condition is no longer hit.

Reply:
According to the [docs](https://github.com/Polymer/polymer/blob/master/lib/elements/dom-repeat.html#L208), `observe` (which sets `__observePaths`) is only relevant when using `sort` and/or `filter` so this seems ok.